pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -1822,7 +1822,7 @@ let
         mktplcRef = {
           publisher = "github";
           name = "copilot-chat";
-          version = "0.14.2024032901";  # compatible with vscode 1.88.1
+          version = "0.14.2024032901"; # compatible with vscode 1.88.1
```

```suggestion
          version = "0.14.2024032901"; # compatible with vscode 1.88.1
```
If you just rebase, you won't have to fix this here.